backend/src/Controllers/voterRolls.controller.ts

```diff
@@ -160,7 +240,12 @@ const getVoterAuth = async (req: any, res: any, next: any) => {
         req.authorized_voter = true;
         if (req.electionRollEntry.length == 0) {
             //Adds voter to roll if they aren't currently
-            const NewElectionRoll = await ElectionRollModel.submitElectionRoll(req.election.election_id, [req.voter_id], false)
+            const history = [{
+                action_type: 'added',
+                actor: req.user?.email || req.voter_id,
```

Comment:
I'd suggest relying on ids rather than email.  Theoretically we'd give users ability to change their emails, which would then break our data.

Reply:
That's a good point, the only issue is we'd want to display the user's email so we'd need some way to look that up.